languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10084,6 +10084,68 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ADJ_F_ET_ADJ_F" name="adj f et adj f">
+          <antipattern>
+              <token postag="N f s"/>
+              <token postag="J f s"/>
+              <token>et</token>
+              <marker>
+                  <token postag="V ppa m s"/>
+              </marker>
+              <token postag="V.*" postag_regexp="yes"/>
+          </antipattern>
+          <antipattern>
+              <token postag="N f s"/>
+              <token postag="J f s"/>
+              <token>et</token>
+              <marker>
+                  <token postag="J m s"/>
+              </marker>
+              <token postag="N m s|N e s" postag_regexp="yes"/>
+          </antipattern>
+          <antipattern>
+              <token postag="N f s"/>
+              <token postag="J f s"/>
+              <token>et</token>
+              <marker>
+                  <token postag="J m s"/>
+              </marker>
+              <token postag="V ppa m s|V ppr" postag_regexp="yes"/>
+          </antipattern>
+          <antipattern>
+              <token postag="N f s" postag_regexp="yes"/>
+              <token postag="J f s" postag_regexp="yes"/>
+              <token>et</token>
+              <marker>
+                  <token postag="J m s" postag_regexp="yes" regexp="yes">.*[er$]|.*[ant$]</token>
+              </marker>
+              <token postag="V.*" postag_regexp="yes"/>
+          </antipattern>
+          <antipattern>
+              <token postag="N f s"/>
+              <token postag="J f s" postag_regexp="yes"/>
+              <token>et</token>
+              <marker>
+                  <token postag="J m s">fait</token>
+              </marker>
+              <token postag="V.*" postag_regexp="yes"/>
+          </antipattern>
+          <pattern>
+              <token postag="N f s"/>
+              <token postag="J f s"/>
+              <token>et</token>
+              <marker>
+                  <token postag="J m s"/>
+              </marker>
+              <token postag="V.*" postag_regexp="yes"/>
+          </pattern>
+          <message>Faux</message>
```

Comment:
Faux ! (Tu connais la chaîne de Norman ?)

This message wasn't ready yet, apparently.